megamek/src/megamek/server/ServerHelper.java

```diff
@@ -395,4 +395,22 @@ else if ((entity.heat >= 14) && !entity.isShutDown()) {
             vPhaseReport.addAll(s.destroyEntity(entity, "pilot death", true));
         }
     }
+    
+    /**
+     * Helper function that causes an entity to sink to the bottom of the water
+     * hex it's currently in.
+     */
+    public static void sinkToBottom(Entity entity) {
+        IHex fallHex = entity.getGame().getBoard().getHex(entity.getPosition());
```

Comment:
Based on our historical bug reports you should probably return early if entity or fallHex is null.